Vignettes/usingcchsflow.Rmd

```diff
@@ -0,0 +1,186 @@
+
+There are many variables in the CCHS that changes in categories between cycles in a way that the variable cannot perserve all categories for all CCHS cycles. An example is the categorical variable for `age`. `cchsflow` offer several options to facilitate the use of these variables with inconsisten categories across cycles.
+
+#### Example 3.1: transform category `age` variable into a common variable for only cycles with the same category responses
+
+The categories in `age` variable in the CCHS changed in 2005 and therefore it is not possible to have the same `age` categories across all CCHS cycels. `cchsflow` offers two optins. The first option is to transform the `age` variable into two variables. `DHHGAGE_A` is the age variable for CCHS cycles 2001-2003, and `DHHGAGE_B` is the age variable for CCHS cycles 2005-2014. With this option, you cannot combine `DHHGAGE_A` and `DHHGAGE_B` into a single dataset.
+
+```{r, warning=FALSE}
+age2001 <- RecWTable(dataSource = cchsMock2001, variableDetails = varDetails, datasetName = "cchs-82M0013-E-2001-c1-1-general-file", log = TRUE, variables = c("DHHGAGE_A"), varLabels = c(DHHGAGE_A = "Age"))
+datatable(age2001, options = list(columnDefs = list(list(className = 'dt-center', targets = 0:1)), dom = 't'))
+
+age2013 <- RecWTable(dataSource = cchsMock2013, variableDetails = varDetails, datasetName = "cchs-82M0013-E-2013-2014-Annual-component", log = TRUE, variables = c("DHHGAGE_B"), varLabels = c(DHHGAGE_B = "Age"))
+datatable(age2013, options = list(columnDefs = list(list(className = 'dt-center', targets = 0:1)), dom = 't'))
+```
+
+
+#### Example 3.2: transform the categorical `age` variable into a continuous `age` variable
+
+The categorical `age` variable can also be transformed into a single continuous `age` variable. This variable takes the midpoint age of each category for all CCHS cycles. With this option, the age category variable from all CCHS cycles can be combined into a single dataset.
+
+```{r, warning=FALSE}
+age2001_cont <- RecWTable(dataSource = cchsMock2001, variableDetails = varDetails, datasetName = "cchs-82M0013-E-2001-c1-1-general-file", log = TRUE, variables = c("DHHGAGE_cont"), varLabels = c(DHHGAGE_cont = "Age"))
+datatable(age2001_cont, options = list(columnDefs = list(list(className = 'dt-center', targets = 0:1)), dom = 't'))
+
+age2013_cont <- RecWTable(dataSource = cchsMock2013, variableDetails = varDetails, datasetName = "cchs-82M0013-E-2013-2014-Annual-component", log = TRUE, variables = c("DHHGAGE_cont"), varLabels = c(DHHGAGE_cont = "Age"))
+datatable(age2013_cont, options = list(columnDefs = list(list(className = 'dt-center', targets = 0:1)), dom = 't'))
+
+combinedAge_cont <- bind_rows(age2001_cont, age2013_cont)
+datatable(combinedAge_cont, options = list(columnDefs = list(list(className = 'dt-center', targets = 0:1)), dom = 't'))
+```
+
+
+### Step 4. Label variables your final dataset
```

Comment:
Could you include
1)  an executable example where labels are added in a single database using RecWTable?
2) an executable example of SetDataLabels(). @DarkShadeKnigh SetDataLabels() is a bllflow function, right? 
The example right now is not executable.

Reply:
2. I believe u might have to pull the recode-with-table branch again because u might have the version before the PR to master prep.

Reply:
The example was not in the format that allows code to be executed. ```{r}

I didn't even try to run it!